apps/api/src/app/groups/groups.service.ts

```diff
@@ -41,6 +41,15 @@ export class GroupsService {
         return await this.groupRepository.find();
     }
 
+    /**
+     * Show admin's groups data in database.
+     * @param adminUserId account userId from jwt auth.
+     * @returns List of admin's existing groups.
+     */
+    async getOnesGroupsData(adminUserId: string): Promise<GroupData[]>{
```

Comment:
I would call it `getGroupsByAdmin`, just to be consistent with the pre-defined repository methods